docs/modules/MongoDb.md

```diff
@@ -8,6 +8,7 @@ To have your database properly cleaned you should configure it to access the dat
 
 In order to have your database populated with data you need a valid js file with data (of the same style which can be fed up to mongo binary)
 File can be generated by RockMongo export command
+You can also use directoy, generated by ```mongodump``` tool or it's ```.tar.gz``` archive (not available for Windows systems), generated by ```tar -czf <archve_file_name>.tar.gz <path_to dump directory>```.
```

Comment:
Actually you done have to update docs, they are generated from the code comments automatically on release.